.github/workflows/smoke_test.yml

```diff
@@ -0,0 +1,43 @@
+name: Smoke Tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+defaults:
+  run:
+    working-directory: ./typescript
+jobs:
+  test:
+    name: Smoke Tests
+    timeout-minutes: 15
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '>= 18'
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Install Test Dependencies
+        run: npm install concurrently http-server wait-on
+
+      - name: Build Storybook
```

Comment:
I guess we need to build the wsc packages first using `nx`. But we could also consider adding the tests to the `typescript.yml` workflow instead, where everything is build anyway, including storybook and the wsc packages